env-check.sh

```diff
@@ -31,28 +34,55 @@ func_check_python_pkg(){
 
 func_check_file(){
     if [ -e "$1" ]; then
-        return
+        return 0
     fi
-    out_str="$out_str""Optional: Enable dynamic debug logs in \e[31m $1 \e[0m file\n\tFor example,\n\toptions snd_sof dyndbg=+p\n\toptions snd_sof_pci dyndbg=+p\n"
+    case $1 in
+        "$dyn_dbg_conf")
+            out_str="$out_str""\tOptional: Enable dynamic debug logs in \e[31m$1\e[0m file\n"
+            out_str="$out_str""\tFor example:\n\t\toptions snd_sof dyndbg=+p\n\t\toptions snd_sof_pci dyndbg=+p\n"
+            ;;
+        "$sof_logger")
+            out_str="$out_str""\tExecutable sof-logger should be installed to \e[31m$1\e[0m\n"
+            ;;
+        "$sof_ctl")
+            out_str="$out_str""\tExecutable sof-ctl should be installed to \e[31m$1\e[0m\n"
+            ;;
+        "$sof_ldc")
+            out_str="$out_str""\tSOF ldc file should be installed to \e[31m$1\e[0m\n"
+            ;;
+    esac
+
     check_res=1
 }
 
-check_res=0
+func_check_return_val() {
+    if [ "$1" -eq 0 ]; then
+        printf "Pass\n"
+    else
+        printf '\e[31mWarning\e[0m\n'
+        # Need ANSI color characters.
+        # shellcheck disable=SC2059
+        printf "$out_str"
+    fi
+}
+
+out_str="" check_res=0
+printf "Checking for required files\t\t"
+func_check_file "$sof_logger"
```

Comment:
For executable binary check, a new func_check_path would be good. Please see above comments